README.md

```diff
@@ -37,10 +37,85 @@ and treat as best.
 A Polkadot [collator](https://wiki.polkadot.network/docs/en/learn-collator) for the parachain is
 implemented by the `polkadot-parachain` binary (previously called `polkadot-collator`).
 
-## Installation
-
+## Installation and Setup
 Before building Cumulus SDK based nodes / runtimes prepare your environment by following Substrate [installation instructions](https://docs.substrate.io/main-docs/install/).
 
+To launch a local network, you can use [zombienet](https://github.com/paritytech/zombienet) for quick setup and experimentation or follow the [manual setup](#manual-setup).
+
+### Zombienet
+We use zombienet to spin up networks for integration tests and local networks. Follow [these installation steps](https://github.com/paritytech/zombienet#requirements-by-provider) to set it up on your machine.
+A simple network specification with two relay chain nodes and one collator is located at `zombienet_examples/simple_network.toml`.
+
+
+#### Which provider should I use?
+Zombienet offers multiple providers to run networks. Choose the one that best fits your needs:
+- **Podman:** Choose this if you want to spin up a network quick and easy.
+- **Native:** Choose this if you want to develop and deploy your changes. Requires compilation of the binaries.
+- **Kubernetes:** Choose this for advanced use-cases or running on cloud-infrastructure.
+
+#### How to run
+To run the example network, use the following commands:
+
+```bash
+# Podman provider
+zombienet --provider podman spawn ./zombienet_examples/simple_network.toml
+
+# Native provider, assumes polkadot and polkadot-parachains binary in $PATH
+zombienet --provider native spawn ./zombienet_examples/simple_network.toml
+```
+
+### Manual Setup
+#### Launch the Relay Chain
+
+```bash
+# Clone
+git clone https://github.com/paritytech/polkadot
+cd polkadot
+
+# Compile Polkadot with the real overseer feature
+cargo build --release --bin polkadot
+
+# Generate a raw chain spec
+./target/release/polkadot build-spec --chain rococo-local --disable-default-bootnode --raw > rococo-local-cfde.json
+
+# Alice
+./target/release/polkadot --chain rococo-local-cfde.json --alice --tmp
+
+# Bob (In a separate terminal)
+./target/release/polkadot --chain rococo-local-cfde.json --bob --tmp --port 30334
+```
+
+#### Launch the Parachain
+
+```bash
+# Clone
+git clone https://github.com/paritytech/cumulus
+cd cumulus
+
+# Compile
+cargo build --release --bin polkadot-parachain
+
+# Export genesis state
+./target/release/polkadot-parachain export-genesis-state > genesis-state
```

Comment:
@skunert 
I am thinking, it would be cool to aligning manual setup with zombinet run maybe,
if you run zombinet and manul setup, it will run the same thing - to avoid confusion :)

small_network.toml is running `statemine-local`
and manual run is running some sample rococo-parachain

but this is just very nice to have

Reply:
Good catch, adjusted to use `rococo-parachain`.